Add Crystal Compass (#944)
crystal-lang/crystal-website: _posts/2025-05-23-crystal-compass.md

Author: I don't get the "steam to team" change. There is no "steam" in the current copy.
And the suggested text with "support team pairs production teams" sounds a bit odd.

---
title: "Crystal Compass"
summary: "The team behind the language, on your team"
author: straight-shoota
categories: community
tags: [Crystal Compass, ecosystem, Manas.Tech]
---

[Manas.Tech](https://manas.tech/), the company behind Crystal, launches a new service:

[**Crystal Compass**](https://manas.tech/projects/crystal/crystal-compass/)
is a paid subscription that offers technical support and code reviews from the
creators of the Crystal language.

> This is a service for companies who use or prepare to use Crystal in production.

Reviewer: This is a service for **people** who use or prepare to use Crystal in production

> This is a service for companies who use or prepare to use Crystal in production.

Reviewer: ```suggestion
This is a service for people who use or prepare to use Crystal in production.
```

Our support service pairs production teams with Crystal experts,

> providing PR reviews, architectural input, language guidance, and critical conversations.

Author: This was taken from the draft in https://outline.manas.tech/doc/crystal-services-sketch-w07i8yJcBY

I presume it refers to constructive criticism, like scrutinizing the code and offering honest advice.

> Its goal is to strengthen Crystal's user community. We want to help those who use

Reviewer: Its goal is to strengthen Crystal's user community, helping those who already use

> Its goal is to strengthen Crystal's user community. We want to help those who use

Reviewer: ```suggestion
Its goal is to strengthen Crystal's user community, helping those who already use
```

> Crystal in production and new adopters by offering a simple and uncomplicated support service.

Reviewer: ```suggestion
Crystal in production and new adopters by offering a simple and practical support service.
```

> This helps the project by securing funding for the people who work on the language,

Reviewer: This aims to make Crystal more attractive as a platform to build successful products,

> This helps the project by securing funding for the people who work on the language,

Reviewer: ```suggestion
This aims to make Crystal more attractive as a platform to build successful products,
```

> and makes it more attractive as a platform to build a successful product.

Reviewer: ```suggestion
while ensuring ongoing language development and maintenance.
```

> and makes it more attractive as a platform to build a successful product.

Reviewer: ```suggestion
This is a service for companies who use or prepare to use Crystal in production.
Our support service pairs production teams with Crystal experts,
providing PR reviews, architectural input, language guidance, and critical conversations.
```
